src/app/examples/toolbar-example/toolbar-example.component.html

```diff
@@ -1,6 +1,6 @@
-<kirby-toolbar title="Toolbar title" [themeColor]="color?.name" (back)="backButtonSelected()">
-  <kirby-icon secondary size="md"></kirby-icon>
-  <kirby-icon end size="md" name="more"></kirby-icon>
+<kirby-toolbar title="Toolbar title" [themeColor]="color?.name" (back)="backButtonSelected()" (primarySelected)="primarySelected()" (secondarySelected)="secondarySelected()">
```

Comment:
Please use this naming for events and event handlers (see: https://angular.io/guide/styleguide#dont-prefix-output-properties):
```suggestion
<kirby-toolbar title="Toolbar title" [themeColor]="color?.name" (back)="onBack()" (primarySelect)="onPrimarySelect()" (secondarySelect)="onSecondarySelect()">
```